Update update.bat

skeith opened the pull request:
Using variable for python version on python command execution so it will work on version higher than 3.5

Have not test it on 3.6 (need tester)
It does not have the ability to stop python 2.x being used but I assume the error from lower python version will make it fail

skeith committed:
Update update.bat

Using variable for python version on python command execution so it will work on version higher than 3.5

Have not test it on 3.6 (need tester)
It does not have the ability to stop python 2.x being used but I assume the error from lower python version will make it fail

Kowlin commented:
Fixes #544, should work with 3.6 and 3.5 due to the magic of *math* being involved.

Jan200101 reviewed (commented):
Code looks good but Im gonna test it now with Python3.6

Jan200101 commented:
Code looks good but Im gonna test it now with Python3.6

Jan200101 reviewed (approved):
echo. doesnt seem to work for some reason and there is a 0 command it says its getting.
But aside of that its good and works.

Will only get a bit complicated if someone got python 3.6 installed but used python3.5 for the bot

skeith commented:
@Jan200101 echo is commented (my vars echo) , they're there for test purposes. I could make this bat scans for installed pythons and make selection but I assume 3.5 will still be mostly used because the guide says so

skeith committed:
Ban2.x Python and Selections

Adding
- Ban Python2.x
- Selection for users with more than one Python installs (which is rare)
- Fallback to AUTO

Twentysix26 commented:
Implemented a less complex solution
https://github.com/Twentysix26/Red-DiscordBot/commit/607f88cf758f839930379946565fff705fbadc64

Jan200101 commented:
what if someone has python 3.2 installed ?

Twentysix26 commented:
The old version didn't prevent this either since it defaulted to `python` as a last attempt. The only difference in this change is that it allows >3.5 python installations.
